lightning-types/src/features.rs

```diff
@@ -177,6 +179,10 @@ mod sealed {
 			ZeroConf | Keysend,
 			// Byte 7
 			Trampoline,
+			// Byte 8 - 31
+			,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
what `,,,,,,,,,,,,,,,,,,,,,,,,` means in this case?

Reply:
These are used by `define_context ` to form `KNOWN_FEATURE_MASK`, which needs bytes 8-31 zeroed.